backend/.env.example

```
@@ -244,7 +244,7 @@ FAN_API_PASSWORD=
 # Optional Sentry DSN for backend error reporting
 SENTRY_DSN=
 SENTRY_SEND_DEFAULT_PII=1
-SENTRY_TRACES_SAMPLE_RATE=0.0
-SENTRY_PROFILES_SAMPLE_RATE=0.0
-SENTRY_ENABLE_LOGS=0
-SENTRY_LOG_LEVEL=error
+SENTRY_TRACES_SAMPLE_RATE=1.0
+SENTRY_PROFILES_SAMPLE_RATE=1.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SENTRY_PROFILES_SAMPLE_RATE key should go before the SENTRY_SEND_DEFAULT_PII key

(UnorderedKey)